pkg/skaffold/deploy/helm.go

```diff
@@ -144,3 +163,39 @@ func (h *HelmDeployer) deleteRelease(out io.Writer, r v1alpha2.HelmRelease) erro
 	out.Write(stdout)
 	return nil
 }
+
+func (h *HelmDeployer) evaluateTemplates(setValueTemplates map[string]string, r *build.BuildResult) (map[string]string, error) {
+	results := map[string]string{}
+	envMap := map[string]string{}
+	for _, env := range environ() {
```

Comment:
Can you re-factor and re-use some of the code here https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/build/tag/env_template.go#L53